lightning/src/routing/network_graph.rs

```diff
@@ -162,17 +162,17 @@ pub enum NetworkUpdate {
 		/// The update to apply via [`NetworkGraph::update_channel`].
 		msg: ChannelUpdate,
 	},
-	/// An error indicating only that a channel has been closed, which should be applied via
-	/// [`NetworkGraph::close_channel_from_update`].
-	ChannelClosed {
+	/// An error indicating only that a channel has failed, which should be applied via
```

Comment:
I think it'd be nice to elaborate on what "failed" means here

Reply:
Done. Let me know if you'd prefer a different elaboration.